lib/queue/worker.py

```diff
@@ -144,4 +144,5 @@ def increment_message_error_counts(self, messages_with_queues: List[Tuple]):
             else:
                 updated_message = schemas.parse_message(message_body)
                 updated_message.retry_count = retry_count
+                queue.delete_messages(Entries=[self.delete_message_entry(message)])
```

Comment:
this isn't including the idx (which is mapped to the id) is it needed?

Reply:
yep - idx is only needed as some unique identifier in a list of items - and on the other side of this, it just defaults to 0 if not passed along, so this works fine!